mypy/util.py

```diff
@@ -824,3 +824,8 @@ def plural_s(s: int | Sized) -> str:
         return "s"
     else:
         return ""
+
+
+def safe(o: T | None) -> T:
```

Comment:
should this take a `message` argument as well

Reply:
Hows that look?

Reply:
```py
def safe(o: T | None, msg: str) -> T:
    ...

print(safe(foo, "foo was None"))
```

also what `o` means? should it be renamed to something more descriptive like `value`?